code/game/machinery/constructable_frame.dm

```diff
@@ -48,6 +48,28 @@
 		D = "Requires [english_list(component_list)]."
 	desc = D
 
+/obj/machinery/constructable_frame/verb/rotate()
+	set name = "Rotate Frame Counter-Clockwise"

```

Comment:
This may be slightly out of scope but this should really be a verb at the obj or machinery level.

Reply:
screeeeeeeee out of scooooooope

Reply:
oh shit so it's already added above lmao